FSharpKoans/AboutAsserts.fs

```diff
@@ -1,38 +1,37 @@
 ﻿namespace FSharpKoans
-open FSharpKoans.Core
-
+open Expecto
 //---------------------------------------------------------------
 // Getting Started
 //
-// The F# Koans are a set of exercises designed to get you familiar 
-// with F#. By the time you're done, you'll have a basic 
+// The F# Koans are a set of exercises designed to get you familiar
+// with F#. By the time you're done, you'll have a basic
 // understanding of the syntax of F# and learn a little more
 // about functional programming in general.
 //
 // Answering Problems
-// 
+//
 // This is where the fun begins! Each Koan method contains
-// an example designed to teach you a lesson about the F# language. 
+// an example designed to teach you a lesson about the F# language.
 // If you execute the program defined in this project, you will get
 // a message that the AssertEquality koan below has failed. Your
 // job is to fill in the blank (the __ symbol) to make it pass. Once
 // you make the change, re-run the program to make sure the koan
-// passes, and continue on to the next failing koan.  With each 
+// passes, and continue on to the next failing koan.  With each
 // passing koan, you'll learn more about F#, and add another
 // weapon to your F# programming arsenal.
 //---------------------------------------------------------------
-[<Koan(Sort = 1)>]
+
 module ``about asserts`` =
 
-    [<Koan>]
-    let AssertExpectation() =
+  let tests =
+    testList "teaching about assertions" [
+      testCase "assert expectation" <| fun () ->

```

Comment:
I'd be curious to see if there are other ways to arrange these tests.  I like assembling the list and enforcing order, but at the same time the forced indentation and wall of tests can be intimidating to a new user.  Maybe separate functions, assembled to a list at the very bottom of the module?